rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -1294,15 +1294,24 @@ void ppu_thread::fast_call(u32 addr, u32 rtoc)
 
 u32 ppu_thread::stack_push(u32 size, u32 align_v)
 {
+	// GPR[1] must always be 16 bytes aligned
+	align_v = std::max<u32>(align_v, 0x10);
```

Comment:
It's not very efficient to do it in stack_push, because there are plenty of cases when allocated variables are smaller (u64) but in reality they would be packed. I suggest to place alignment fixup in PPUCallback.h